lib/src/app.dart

```diff
@@ -80,7 +80,7 @@ class AppConfiguration {
   /// The [HttpClient] that will be used for HTTP requests during authentication.
   ///
   /// You can use this to override the default http client handler and configure settings like proxies,
-  /// client certificates, and cookies. While these are not required to connect to MongoDB Realm under
+  /// client certificates, and cookies. While these are not required to connect to MongoDB Atlas under
```

Comment:
```suggestion
  /// client certificates, and cookies. While these are not required to connect to Atlas App Services under
```

We don't connect to atlas directly - it's always via AAS, even when using remote mongo and functions.